improver/utilities/load.py

```diff
@@ -82,18 +80,12 @@ def load_cube(filepath, constraints=None, no_lazy_load=False,
             If True, bypass cube deferred (lazy) loading and load the whole
             cube into memory. This can increase performance at the cost of
             memory. If False (default) then lazy load.
-        allow_none (bool):
-            If True, when the filepath is None, returns None.
-            If False, normal error handling applies.
-            Default is False.
 
     Returns:
-        iris.cube.Cube:
-            Cube that has been loaded from the input filepath given the
+        iris.cube.CubeList:
+            CubeList that has been created from the input filepath given the
             constraints provided.
     """
-    if filepath is None and allow_none:
-        return None
     # Remove metadata prefix cube if present
```

Comment:
```suggestion
    # Remove legacy metadata prefix cube if present
```
Some reference to the fact this is no more?